src/features/AddAccount/AddAccountFlow.tsx

```diff
@@ -93,6 +93,7 @@ const AddAccountFlow = withRouter(({ history, match }) => {
       const handleAddition =
         accountData.length === 1
           ? () => {
+              if (accountData.length > 1) displayNotification(NotificationTemplates.saveSettings);
```

Comment:
If we test for the length to be equal to 1, why do we test to see if it's greater than one after ?

Reply:
> If we test for the length to be equal to 1, why do we test to see if it's greater than one after ?

We want to display the notification to save settings after the user adds their second account because that's the point where it starts to get annoying to re-add your accounts.

Reply:
nice catch xD

Reply:
I think it should be greater or equal to 1 ;)